src/python/py/models/builder.py

```diff
@@ -1996,6 +2003,7 @@ def get_args():
                 enable_cuda_graph = 1 : The model can use CUDA graph capture for CUDA execution provider. If enabled, all nodes being placed on the CUDA EP
                     is the prerequisite for the CUDA graph to be used correctly. It is not guaranteed that cuda graph be enabled as it depends on the model
                     and the graph structure.
+                quant_provider = {inc, modelopt} : Quantization provider
```

Comment:
Can you add some more descriptive information about the `quant_provider` options here? You can look at the above options for description examples.

Can you also add examples in the [README](https://github.com/microsoft/onnxruntime-genai/blob/main/src/python/py/models/README.md) as a new section under "Extra Options" after "Enable Cuda Graph"?